tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -0,0 +1,182 @@
+
+
+@parametrize('name', **valid_data_list('ui'))
+def test_positive_create_libvirt(session, name):
+    libvirt_url = settings.compute_resources.libvirt_hostname
+    with session:
+        session.computeresource.create({
+            'name': name,
+            'description': gen_string('alpha'),
+            'provider': 'Libvirt',
+            'provider_content.url': libvirt_url,
+            'provider_content.display_type': 'VNC',
+            'provider_content.console_passwords': True,
+        })
+        assert session.computeresource.search(name)[0]['Name'] == name
+
+
+@parametrize('name', **valid_data_list('ui'))
+def test_positive_create_vmware(session, name):
+    vmware_vcenter = settings.vmware.vcenter
+    vmware_user = settings.vmware.username
+    vmware_password = settings.vmware.password
+    with session:
+        session.computeresource.create({
+            'name': name,
+            'description': gen_string('alpha'),
+            'provider': 'VMware',
+            'provider_content.vcenter': vmware_vcenter,
+            'provider_content.user': vmware_user,
+            'provider_content.password': vmware_password,
+        })
+        assert session.computeresource.search(name)[0]['Name'] == name
+
+
+@parametrize('name', **valid_data_list('ui'))
+def test_positive_create_rhv(session, name):
+    rhev_url = settings.rhev.hostname
+    username = settings.rhev.username
+    password = settings.rhev.password
+    with session:
+        session.computeresource.create({
+            'name': name,
+            'description': gen_string('alpha'),
+            'provider': 'RHV',
+            'provider_content.url': rhev_url,
+            'provider_content.user': username,
+            'provider_content.password': password,
+        })
+        assert session.computeresource.search(name)[0]['Name'] == name
+
+
+def test_positive_rename(session):
+    name = gen_string('alpha')
+    ak_name = gen_string('alpha')
+    docker_url = settings.docker.external_url
+    with session:
+        session.computeresource.create({
+            'name': name,
+            'description': gen_string('alpha'),
+            'provider': 'Docker',
+            'provider_content.url': docker_url,
+        })
+        session.computeresource.edit(name, {
+            'name': ak_name,
+        })
+        assert session.computeresource.search(ak_name)[0]['Name'] == ak_name
+
+
+def test_positive_delete(session):
+    name = gen_string('alpha')
+    docker_url = settings.docker.external_url
+    with session:
+        session.computeresource.create({
+            'name': name,
+            'description': gen_string('alpha'),
+            'provider': 'Docker',
+            'provider_content.url': docker_url,
+        })
+        session.computeresource.delete(name)
+        assert not session.computeresource.search(name)
+
+
+@parametrize('name', **valid_data_list('ui'))
+def test_positive_rhev_v3(session, name, version=3):
```

Comment:
1) why different approaches for edit and create?
2) `'provider_content.api4': True,` for both? It should be False for v3 and True for v4
3) same for asserts, use simple True and False for corresponding situations